src/pudl/output/new_ferc1.py

```diff
@@ -14,12 +18,249 @@ def denorm_plants_utilities_ferc1(
     return pd.merge(plants_ferc1, utilities_ferc1, on="utility_id_ferc1")
 
 
+@asset(io_manager_key="pudl_sqlite_io_manager", compute_kind="Python")
+def denorm_plants_steam_ferc1(
+    denorm_plants_utilities_ferc1: pd.DataFrame, plants_steam_ferc1: pd.DataFrame
+) -> pd.DataFrame:
+    """Select and joins some useful fields from the FERC Form 1 steam table.
+
+    Select the FERC Form 1 steam plant table entries, add in the reporting
+    utility's name, and the PUDL ID for the plant and utility for readability
+    and integration with other tables that have PUDL IDs.
+    Also calculates ``capacity_factor`` (based on ``net_generation_mwh`` &
+    ``capacity_mw``)
+
+    Args:
+        denorm_plants_utilities_ferc1: Denormalized dataframe of FERC Form 1 plants and
+            utilities data.
+        plants_steam_ferc1: The normalized FERC Form 1 steam table.
+
+    Returns:
+        A DataFrame containing useful fields from the FERC Form 1 steam table.
+    """
+    steam_df = (
```

Comment:
It would be nice to share the code between this code and the old code, unless we decided that we are deleting the old code right now!

I think we can do this by just calling the asset in the old code:

```
@asset(...)
def denorm_plants_steam_ferc1(denorm_plants_utilities_ferc1, plants_steam_ferc1):
    # do cool stuff

def plants_steam_ferc1(...)
    "Deprecation warning, yada yada"
    # read input tables from SQL
    denorm_plants_steam_ferc1(... input tables that we just read above)
```

Might make it a little easier to have confidence in the code going forward - but if we're just gonna delete the old functions altogether then this whole point is moot.

Reply:
We've deleted these!